dlt/extract/pipe.py

```diff
@@ -358,7 +359,7 @@ def _verify_head_step(self, step: TPipeStep) -> None:
         if not isinstance(step, (Iterable, Iterator, AsyncIterator)) and not callable(step):
             raise CreatePipeException(
                 self.name,
-                "A head of a resource pipe must be Iterable, Iterator, AsyncIterator or a Callable",
+                "A head of a resource pipe must be: [Iterable, Iterator, AsyncIterator, Callable]",
```

Comment:
praise: I agree with [...] bein more user-friendly than ",".join([...]), because I like bracketing for the implicit context